index.html

```diff
@@ -289,6 +290,37 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <section class="contact flex" id="contact">
+        <p class="parag text-dark">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+        <form
+          class="flex"
+          action="https://formspree.io/f/xdobralq"
+          method="post"
+          id="mail"
+        >
+          <input
+            name="fname"
+            id="name"
+          />
+          <input
+            name="email"
+            type="email"
+            type="text"
+            required
+            placeholder="Email address"
+            id="email"
+          />
+          <textarea
+            name="message"
```

Comment:
- [OPTIONAL] Same as here, optional because the previous review forgot to mention it, but please remember to add the `max length of `500` characters as we move onto the next phase of the project.